zephyr/CMakeLists.txt

```diff
@@ -24,6 +24,12 @@ if(CONFIG_OPENTHREAD_SLAAC)
   set(OT_SLAAC ON CACHE BOOL "Enable SLAAC")
 endif()
 
+set(OT_OUI "${CONFIG_OPENTHREAD_STACK_VENDOR_OUI}" CACHE STRING "OpenThread vendor OUI address")
```

Comment:
I don't see `OT_OUI` and `OT_CUSTOM_EUI64` options in OpenThread, are they going to be added in the near future?